projects/kit/components/data-list-wrapper/data-list-wrapper.component.ts

```diff
@@ -55,17 +60,17 @@ export class TuiDataListWrapperComponent<T> implements TuiDataListAccessor<T> {
 
     constructor(
         @Inject(TUI_ITEMS_HANDLERS)
-        private readonly itemsHandlers: TuiItemsHandlers<T>,
+        private readonly itemsHandlers: TuiItemsHandlers<T | T[]>,
```

Comment:
Items handlers only work with `T`, not with `T[]`

Reply:
fixed